specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -172,6 +204,24 @@ The actual dockerfile can be found at [Dockerfile](/projects/frontend/cicd/Docke
 New versions are released by changing the version in [version.txt](/projects/frontend/cicd/version.txt)
 New releases are published under the image name [registry.hub.docker.com/versatiledatakit/vdk-cicd-base-gui](https://hub.docker.com/r/versatiledatakit/vdk-cicd-base-gui)
 
+#### Release
+
+Frontend packages are published under the
+[@verstiledatakit](https://www.npmjs.com/settings/versatiledatakit/packages)
+namespace in npm registry. Merging changes to `@versatiledatakit/shared` into
+`main` versions the package and publishes it to the npm registry. Merging
+changes to `@versatiledatakit/data-pipelines` into `main` versions both
+data-pipelines and shared packages and publishes them to the npm registry.
+
+Versioning for each package is based on its `version.txt` file and the gitlab
+pipeline id. Major and minor versions are taken from `version.txt`. The gitlab
+pipeline id, which is guaranteed to be unique, serves as the patch version. Note
+that `package.json` files do not represent package versions and should not be
+used as a source of truth for versioning.
```

Comment:
Rather missing instructions on working with Frontend release versioning.
For example, how to bump a patch, minor and major version?

nit

may add a link to contributing guide that states semver is applied.